src/__test__/actions/session.actions.test.js

```diff
@@ -0,0 +1,45 @@
+// third-party libraries
+import expect from 'expect';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+// constants
+import constants from '../../_constants';
+
+// actions
+import { expireSession } from '../../_actions/session.action';
+
+// mock data
+let isSessionExpired = false;
+
+const {
+  SESSION_EXPIRED
+} = constants;
+
+// store
+const middleware = [thunk];
+const mockStore = configureMockStore(middleware);
+let store;
+
+afterEach(() => {
+  store.clearActions();
+});
+
+describe('Session action when a session is expired', () => {
+  store = mockStore({});
+  it('dispatches SESSION_EXPIRED type and sets isSessionExpired to true', () => {
+    isSessionExpired = true;
+    store.dispatch(expireSession(isSessionExpired));
+    expect(store.getActions()[0].type).toEqual(SESSION_EXPIRED);
```

Comment:
A recommendation for future tests:
- Instead of having `expect(store.getActions()[0].type).toEqual(SESSION_EXPIRED)`, you can use the jest's `toContainEquals` for matching. Therefore, you'll have something like:
```
const actionCreator = {
  type: MY_CONSTANT,
  payload: 'My_payload'
};
expect(store.getActions()).toContainEquals(actionCreator);
```
This way, you don't have to keep track of the action index in `getActions()`